scanomatic/data_processing/growth_phenotypes.py

```diff
@@ -363,7 +363,7 @@ def generation_time_when(
     linregress_extent: int,
     **kwargs,
 ) -> float:
-    pos = index + linregress_extent
+    pos = int(index + linregress_extent)
```

Comment:
Why do you need `int` here when both `index` and `linregress_extent` are int?

Reply:
I guess it was a lazy fix but without casting I got error
```
scanomatic_1  | Traceback (most recent call last):
scanomatic_1  |   File "/usr/local/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
scanomatic_1  |     self.run()
scanomatic_1  |   File "/usr/local/lib/python3.9/site-packages/scanomatic/server/rpcjob.py", line 139, in run
scanomatic_1  |     next(effector_iterator)
scanomatic_1  |   File "/usr/local/lib/python3.9/site-packages/scanomatic/server/phenotype_effector.py", line 134, in __next__
scanomatic_1  |     self._progress = next(self._phenotype_iterator)
scanomatic_1  |   File "/usr/local/lib/python3.9/site-packages/scanomatic/data_processing/phenotyper.py", line 642, in iterate_extraction
scanomatic_1  |     for x in self._calculate_phenotypes():
scanomatic_1  |   File "/usr/local/lib/python3.9/site-packages/scanomatic/data_processing/phenotyper.py", line 1110, in _calculate_phenotypes
scanomatic_1  |     phenotypes[phenotype][id0, id1] = phenotype(
scanomatic_1  |   File "/usr/local/lib/python3.9/site-packages/scanomatic/data_processing/growth_phenotypes.py", line 640, in __call__
scanomatic_1  |     return generation_time_when(
scanomatic_1  |   File "/usr/local/lib/python3.9/site-packages/scanomatic/data_processing/growth_phenotypes.py", line 377, in generation_time_when
scanomatic_1  |     return flat_times[pos]
scanomatic_1  | IndexError: only integers, slices (`:`), ellipsis (`...`), numpy.newaxis (`None`) and integer or boolean arrays are valid indices
```

Reply:
In that case it looks like the problem is further up the line. Does mypy give any hints?

Reply:
I found the culprit and fixed it